app/src/pages/prompt/PromptVersionTagsConfigCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo } from "react";
+import { graphql, useFragment } from "react-relay";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+
+import { Card } from "@arizeai/components";
+
+import { Icon, Icons } from "@phoenix/components";
+import { tableCSS } from "@phoenix/components/table/styles";
+
+import { PromptVersionTagsConfigCard_data$key } from "./__generated__/PromptVersionTagsConfigCard_data.graphql";
+import { DeletePromptVersionTagButton } from "./DeletePromptVersionTagButton";
+
+export function PromptVersionTagsConfigCard({
+  prompt,
+}: {
+  prompt: PromptVersionTagsConfigCard_data$key;
+}) {
+  const data = useFragment(
+    graphql`
+      fragment PromptVersionTagsConfigCard_data on Prompt {
+        id
+        versionTags {
+          id
+          name
+          description
+        }
+      }
+    `,
+    prompt
+  );
+
+  const columns = useMemo(
+    (): ColumnDef<(typeof tableData)[number]>[] => [
+      {
+        header: "Name",
+        accessorKey: "name",
+      },
+      {
+        header: "Description",
+        accessorKey: "description",
+      },
+      {
+        id: "actions",
+        header: "",
+        size: 10,
+        accessorKey: "id",
+        cell: ({ row }) => {
+          return (
+            <DeletePromptVersionTagButton
+              promptVersionTagId={row.original.id}
+              promptId={row.original.promptId}
+              // We set the key here because aria-components will keep the dialog open on the row that replaces it after delete
+              key={row.original.id}
+            />
+          );
+        },
+      },
+    ],
```

Comment:
Can we right-align these in the table row? Looks a bit odd at the left of the cell

Reply:
Yup, will do